client/components/plans/plan-price/index.jsx

```diff
@@ -55,7 +55,9 @@ const PlanPrice = React.createClass( {
 			return <div className="plan-price is-placeholder" />;
 		}
 
-		if ( abtest( 'planPricing' ) === 'monthly' && plan.raw_price !== 0 ) {
+		if ( ! plan ) {
+			periodLabel = this.translate( 'Loading' );
```

Comment:
We should just return null here, otherwise we get a double-loading indicator.

![double-loading](https://cloudup.com/cuLYC55prTY+)